examples/python/human_pose_tracking/README.md

```diff
@@ -16,9 +17,227 @@ channel = "main"
+
+# Run
 
 ```bash
 pip install -r examples/python/human_pose_tracking/requirements.txt
 python examples/python/human_pose_tracking/main.py
 ```
+# Usage
+
+CLI usage help is available using the `--help` option:
+
+```bash
+$ python examples/python/gesture_detection/main.py --help
+usage: main.py [-h] [--video {backflip,soccer}] [--dataset-dir DATASET_DIR] [--video-path VIDEO_PATH] [--no-segment] [--max-frame MAX_FRAME] [--headless] [--connect]
+               [--serve] [--addr ADDR] [--save SAVE] [-o]
+
+Uses the MediaPipe Pose solution to track a human pose in video.
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --video {backflip,soccer}
+                        The example video to run on.
+  --dataset-dir DATASET_DIR
+                        Directory to save example videos to.
+  --video-path VIDEO_PATH
+                        Full path to video to run on. Overrides `--video`.
+  --no-segment          Don t run person segmentation.
+  --max-frame MAX_FRAME
+                        Stop after processing this many frames. If not specified, will run until interrupted.
+  --headless            Don t show GUI
+  --connect             Connect to an external viewer
+  --serve               Serve a web viewer (WARNING: experimental feature)
+  --addr ADDR           Connect to this ip:port
+  --save SAVE           Save data to a .rrd file at this path
+  -o, --stdout          Log data to standard output, to be piped into a Rerun Viewer
+
+```
+
+# Logging Data
+
+## Timelines for Video
+
+You can utilize Rerun timelines' functions to associate data with one or more timelines. 
+As a result, each frame of the video can be linked with its corresponding timestamp.
+Here is achieved using the  `set_time_seconds`  and  `set_time_sequence`  Rerun's functions.
+
+```python
+
+def track_pose(video_path: str, *, segment: bool, max_frame_count: int | None) -> None:
+    mp_pose = mp.solutions.pose
+
+    # ... existing code ...
+
+    with closing(VideoSource(video_path)) as video_source, mp_pose.Pose(enable_segmentation=segment) as pose:
+        for idx, bgr_frame in enumerate(video_source.stream_bgr()):
+            if max_frame_count is not None and idx >= max_frame_count:
+                break
+
+            rgb = cv2.cvtColor(bgr_frame.data, cv2.COLOR_BGR2RGB)
+            
+            # Associate frame with the data
+            rr.set_time_seconds("time", bgr_frame.time)
+            rr.set_time_sequence("frame_idx", bgr_frame.idx)
+            
+            # ... logging data ...
+
+```
+
+## Keypoint-Connections
+
+The class description contains the information which maps keypoint ids to labels and how to connect
+the keypoints to a skeleton. In both 2D and 3D points, specifying connections between points is essential. 
+Defining these connections automatically renders lines between them. Using the information provided by MediaPipe, 
+you can get the pose points connections from the `POSE_CONNECTIONS` set.
```

Comment:
I think it would be more useful if this would say what kind of datastructure is contained in POSE_CONNECTIONS.